Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.5.1
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+      sqlite3
```

Comment:
I think that we will want to use postgres here instead of sqlite, its not a huge foot print application, and we can keep development in parity with production